openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```diff
@@ -86,6 +87,7 @@
     private final Builder builder;
     private final OpenApiConfig openAPIConfig;
 
+    private OpenAPI model = null;
```

Comment:
Can you move the call to `initializeOpenAPIDocument` into the ctor and make this final? 

If it does need to be deferred, the explicit setting of `null` here is of course redundant and slightly smelly; It may not be strictly necessary, but you might consider using a  `final AtomicReference` to hold the model.

Reply:
I've reworked things a bit to do all initialization in the constructor, which allows `model` to be `final` and eliminates the need to store the builder at all.